docs/generated/cli/create-nx-workspace.md

```diff
@@ -25,6 +25,12 @@ Default: `false`
 
 Show all prompts
 
+### appDir
```

Comment:
This doesn't seem to have been resolved?

Reply:
Updated, `yarn doc` didn't update it could've been a cache thing